src/backend/egl/mod.rs

```diff
@@ -65,10 +65,12 @@ impl ::std::error::Error for EglExtensionNotSupportedError {}
 /// Returns the address of an OpenGL function.
 ///
 /// Result is independent of displays and does not guarantee an extension is actually supported at runtime.
-pub unsafe fn get_proc_address(symbol: &str) -> *const c_void {
-    let addr = CString::new(symbol.as_bytes()).unwrap();
-    let addr = addr.as_ptr();
-    ffi::egl::GetProcAddress(addr) as *const _
+pub fn get_proc_address(symbol: &str) -> *const c_void {
+    unsafe {
+        let addr = CString::new(symbol.as_bytes()).unwrap();
+        let addr = addr.as_ptr();
+        ffi::egl::GetProcAddress(addr) as *const _
+    }
```

Comment:
Is it though? No way this can error?

Reply:
The return type is void, and the doc do not mention any possible errors (Nor can I find any mention through googling). Apart from some sort of undocumented thread killing error handling within that I'm unaware of, its as safe as any of our other "safe" wrappers.

Reply:
:+1: